Add option to disable sessionState in Accordion (#3342)
alphagov/govuk-frontend: CHANGELOG.md

This was added in [pull request #3187: Add disabled styles for form controls](https://github.com/alphagov/govuk-frontend/pull/3187).

> #### Stop the Accordion from remembering and restoring sessions

Reviewer: A thought – we might want to reword this as we already have a few other entries in the changelog that start with stop where we _want_ people to stop doing something, but in this instance we're just _allowing_ people to disable it where it's not desired behaviour:

- [We want you to] stop using the deprecated IE8 mixins and settings
- [We want you to] stop using the `govuk-button--disabled` class on buttons
- [You now have the option to] stop the Accordion from remembering and restoring sessions

Maybe something like 'Configure whether the Accordion remembers and restores sessions'?

Author: Makes sense to me! 

I'm working under the assumption that the changelog is going to get tidied up a bit pre-release anyway, but no harm in making things clearer now.

> By default, the [Accordion](https://design-system.service.gov.uk/components/accordion/) will remember which sections a user has expanded and collapsed upon leaving a page. If the user later returns to the page, this layout will be restored, overriding any sections that have been manually set as `expanded` in code.

Reviewer 2: ```suggestion
By default, when a user leaves a page, the [Accordion](https://design-system.service.gov.uk/components/accordion/) will remember the layout of expanded and collapsed sections selected by the user. If the user returns to the page, this layout will be restored and override any sections manually set as `expanded` in code.
```

> You can now disable this functionality by using the `rememberExpanded` option in the `govukAccordion` Nunjucks macro.

Reviewer 2: Do we need to add a link to the relevant nunjucks macro?

Author: I don't think so. The user-facing version of the Nunjucks macro code is already part of the design system page linked in the previous paragraph.

Reviewer 2: ok great
